CONTRIBUTING.rst

```diff
@@ -161,6 +166,12 @@ The MLflow UI is written in JavaScript. ``npm`` is required to run the Javascrip
 You can verify that ``npm`` is on the PATH by running ``npm -v``, and
 `install npm <https://www.npmjs.com/get-npm>`_ if needed.
 
+Install Node Module Dependencies
+++++++++++++++++++++++++++++++++
+
+.. code-block:: bash
+   brew install pixman cairo pango jpeg
```

Comment:
Can we clarify that this will only work on OSX systems and that these packages need to be sourced from elsewhere for Windows / Linux-based systems?

Reply:
Yeah I will add a note on that